### PR TITLE
docs: adds getting started guide

### PR DESCRIPTION
**Pull request by afzal442**
* Part of #480

### CODE REVIEW

**Review by Souvikns — CHANGES_REQUESTED** [review]
@afzal442 left some comments, I have a question:
what is `glee-template.md` and why do we need it?

File: docs/getting-started.md [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+We will not execute the application and carry out testing with Postman to ensure that it is functioning as intended.
+
+Now to run your glee application, just run:
+
+```
+npm run dev
+# or 
+npm run start
+```
+To send a WebSocket request with a payload e.g. `{"name":"souvik", "time": "1567906535"}` to `ws://localhost:3000/greet`, open Postman and checkout the endpoint:
```
[the remark this answers is not in the document]
Comment (afzal442) [reply docs/getting-started.md:130]: while working on it myself, I found it lgtm. and ultimately I am pointing to your github repo for reference. let's keep it as is. 😄

File: docs/glee-template.md [hunk 2]
```diff
@@ -0,0 +1,89 @@
+---
+title: "Create AsyncAPI Glee template"
+weight: 170
+---
+This tutorial teaches you how to create a simple glee template. You'll use the AsyncAPI Glee template you develop to generate Javascript code. Additionally, you'll create a template code with a reusable component to reuse the custom functionality you create and test your code using an WS server.
+
+<CodeBlock>
```
[the remark this answers is not in the document]
Comment (afzal442) [reply docs/glee-template.md:7]: @Souvikns I think it is github .md that can't render it.
e.g.
https://github.com/asyncapi/website/blob/master/pages/docs/tutorials/getting-started/hello-world.md

File: docs/getting-started.md [hunk 3]
```diff
@@ -0,0 +1,142 @@
+---
+title: Getting Started
+weight: 80
+---
+
+## Introduction
+
```
Comment (afzal442) [inline docs/getting-started.md:7]: @Souvikns I removed that intro part from that other PR and added it here like nextjs docs follows

**Comment by afzal442** [comment]
> what is `glee-template.md` and why do we need it?

that is separate file which gives clear understanding of glee-template step by step like [this](https://github.com/asyncapi/website/blob/master/pages/docs/tutorials/getting-started/hello-world.md).

File: docs/getting-started.md [hunk 4]
```diff
@@ -0,0 +1,142 @@
...
+We will not execute the application and carry out testing with Postman to ensure that it is functioning as intended.
+
+Now to run your glee application, just run:
+
+```
+npm run dev
+# or 
+npm run start
+```
+To send a WebSocket request with a payload e.g. `{"name":"souvik", "time": "1567906535"}` to `ws://localhost:3000/greet`, open Postman and checkout the endpoint:
```
Comment (Souvikns) [inline docs/getting-started.md:138]:
```suggestion
To send a WebSocket request with a payload e.g. `{"name":"SomeOtherName", "time": "1567906535"}` to `ws://localhost:3000/greet`, open Postman and checkout the endpoint:
```

Everything else is fine, just remove my name I think you followed my example which is good, but remove my name 🤣. Remove it from the image as well, please.

**Review by Souvikns — CHANGES_REQUESTED** [review]
Good work @afzal442, just update the image and example and we can merge this.

**Review by Souvikns — APPROVED** [review]
Good job @afzal442 👍🏼

**Pull request merged**